examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,130 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples;
+
+import glide.api.GlideClient;
+import glide.api.logging.Logger;
+import glide.api.models.configuration.GlideClientConfiguration;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.TimeoutException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class StandaloneExample {
+
+    /**
+     * Creates and returns a GlideClient instance. This function initializes a
+     * GlideClient with the provided list of nodes.
+     *
+     * @return A <code>GlideClient</code> connected to the discovered nodes.
+     */
+    public static GlideClient createClient() throws ExecutionException, InterruptedException {
+        String host = "localhost";
+        Integer port = 6379;
+
+        // Check GlideClientConfiguration for additional options.
+        GlideClientConfiguration config =
+                GlideClientConfiguration.builder()
+                        .address(NodeAddress.builder().host(host).port(port).build())
+                        .build();
+
+        try {
+            GlideClient client = GlideClient.createClient(config).get();
+            return client;
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Failed to create client: " + e.getMessage());
```

Comment:
if you're catching the exception here and you don't throw it, the exceptions won't be handled in appLogic

Reply:
please use the same error handling we did in python, where all errors are being handled only in the appLogic loop